app/views/partner_requests/_item.html.erb

```diff
@@ -1,5 +1,5 @@
 <tr>
-  <td><%= form.select :name, POSSIBLE_ITEMS.slice(*@valid_items).invert.collect, {include_blank: 'Select an item'}, {class: 'form-control'} %></td>
+  <td><%= form.select :name, POSSIBLE_ITEMS.slice(*@valid_items).invert.sort_by{|k,v|k}.collect, {include_blank: 'Select an item'}, {class: 'form-control'} %></td>
```

Comment:
Are you sure we need the .collect call there? I think it may be redundant.

Reply:
How do i upload my changes to this pull request?